public/_locales/en/messages.json

```
@@ -1,10 +1,10 @@
 {
   "extensionDescription": {
     "description": "Extension description",
-    "message": "Chrome extension boilerplate developed with Vite, React and Typescript"
+    "message": "안녕! 나는 깃식이야! 나는 무뚝뚝한 감자칩을 좋아해"
```

[제안]
기능에 대한 내용도 추가되었으면 좋겠네요
```suggestion
    "message": "안녕! 나는 깃식이야! 나는 무뚝뚝한 감자칩을 좋아해. 깃허브를 더 편하게 쓸 수 있게 도와줄게!"
```